packages/block-editor/src/components/block-canvas/style.scss

```diff
@@ -2,12 +2,17 @@ iframe[name="editor-canvas"] {
 	width: 100%;
 	height: 100%;
 	display: block;
-}
 
-iframe[name="editor-canvas"]:not(.has-editor-padding) {
-	background-color: $white;
+	&:not(.has-editor-padding):not(.is-zoomed-out) {
+		background-color: $white;
+	}
+
+	&.is-zoomed-out {
+		background-color: $gray-300;
```

Comment:
What if we consistently apply this grey background? Why does it need to be white for non zoom out?

Reply:
Fair question.

Reply:
Isolated views also use the gray background; I don't think white is used anywhere on its own.

Reply:
Thanks for checking! Why can't we add the background for `.has-editor-padding`? Might be worth commenting.

Reply:
Ok I went back to figure out why we need it, and it comes from this PR https://github.com/WordPress/gutenberg/pull/57631 and the white color comes from this other one https://github.com/WordPress/gutenberg/pull/57330. So in reality right now we could keep the gray background for all cases, I think